tests/namespace/test_utilities_namespace.py

```diff
@@ -0,0 +1,81 @@
+import importlib
+import pathlib
+
+import pytest
+
+from pyvista.core.errors import PyVistaDeprecationWarning
+
+namesapce_data = pathlib.Path(__file__).parent / 'namespace-utilities.txt'
+with open(namesapce_data) as f:
+    namespace = f.read().splitlines()
+    # ignore commented data
+    namespace = [n.split(', ')[0] for n in namespace if not n.startswith('#')]
+
+
+@pytest.mark.parametrize('name', namespace)
+def test_utilities_namespace(name):
+    with pytest.warns(PyVistaDeprecationWarning):
+        utilities = importlib.import_module('pyvista.utilities')
+        assert hasattr(utilities, name)
+
+
+@pytest.mark.parametrize(
+    'name',
+    [
+        'algorithms',
+        'arrays',
+        'cell_type_helper',
+        'cells',
+        'common',
+        'docs',
+        'errors',
+        'features',
+        'fileio',
+        'geometric_objects',
+        'helpers',
+        'misc',
+        'parametric_objects',
+        'reader',
+        'regression',
+        'sphinx_gallery',
+        'transformations',
+        'wrappers',
+        'xvfb',
+    ],
+)
+def test_utilities_modules(name):
+    # Smoke test to make sure same modules still exist
+    importlib.import_module(f'pyvista.utilities.{name}')
```

Comment:
Perhaps you can check that the warning is raised and parse the warning message for the proper path to the module?